packages/infolists/src/Components/KeyValueEntry.php

```diff
@@ -15,6 +16,23 @@ class KeyValueEntry extends Entry
 
     protected string | Closure | null $valueLabel = null;
 
+    protected string | Closure | null $emptyLabel = null;
+
+    public function getState(): mixed
+    {
+        $state = parent::getState();
+
+        if ($state === null) {
+            return collect();
+        }
+
+        if (is_array($state)) {
+            return collect($state);
+        }
+
+        return $state;
+    }
```

Comment:
Could you please explain what this override is being used for? I think we should avoid it if possible

Reply:
@danharrin Of course, for some reason the state of the casted array attribute is being converted to `null`, thus causing the @foreach to fail within the template.
I tried my best to track where the value of the state being overwritten and set to `null` when it's `[]`
The proposed override attempts to change the state to `[]` when it's null.

You might have a good point about avoiding such an override since the template now checks for the state value before attempting the loop.
```php
@if($getState())
```

Reply:
`@if (filled($state = ($getState() ?? [])))`, `@foreach ($state ...`

Reply:
@danharrin yeah sounds better. What's your take on the empty state template change?

Reply:
What about it? My only comment would be renaming label to message, probably